lib/config.js

```diff
@@ -45,5 +45,11 @@ export let config = {
                       details.`,
         type: 'boolean',
         default: false,
+    },
+    ignoredFiles: {
+      title: "Ignore files with these extensions",
+      description: "Don't show files with the given extensions.",
+      type: 'string',
```

Comment:
Atom allows for [array config values](https://atom.io/docs/api/v1.22.1/Config#array), which would be appropriate here.

Reply:
Changed to array of string